<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="scribo::demo::shared::image_viewer" name="viewer_" />
      <widget class="QTextEdit" name="text_" />
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="run_btn" >
        <property name="text" >
         <string>&amp;Run</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open" />
    <addaction name="separator" />
    <addaction name="action_Quit" />
   </widget>
   <widget class="QMenu" name="menuWindow" >
    <property name="title" >
     <string>&amp;Dialogs</string>
    </property>
    <addaction name="actionOptions" />
    <addaction name="actionAdvanced_options" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menuWindow" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QDockWidget" name="options_dock_" >
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="floating" >
    <bool>false</bool>
   </property>
   <property name="features" >
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas" >
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle" >
    <string>Options</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" name="verticalLayout_7" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QCheckBox" name="removeBg" >
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>Remove background (slow)</string>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="unskew" >
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>Unskew</string>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="removeNoise" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>Remove noise</string>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="improveContrast" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>Improve brightness/contrast</string>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="advanced_dock_" >
   <property name="font" >
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="windowTitle" >
    <string>Advanced options</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3" >
    <layout class="QVBoxLayout" name="verticalLayout_8" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QCheckBox" name="reduceSize" >
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>Run on subsampled image</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6" >
       <item>
        <widget class="QLabel" name="binarizeLbl" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text" >
          <string>Binarization method:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="binarizationCBox" >
         <property name="font" >
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="currentIndex" >
          <number>2</number>
         </property>
         <item>
          <property name="text" >
           <string>Global threshold</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Local threshold</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Local threshold multi-scale</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_Open" >
   <property name="text" >
    <string>&amp;Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Quit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionDocument_Type" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Document Type</string>
   </property>
  </action>
  <action name="actionOptions" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Options</string>
   </property>
  </action>
  <action name="actionAdvanced_options" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="text" >
    <string>Advanced options</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>scribo::demo::shared::image_viewer</class>
   <extends>QGraphicsView</extends>
   <header location="global" >shared/src/image_viewer.hh</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>332</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
